client/reader/following-stream/_style.scss

```diff
@@ -373,3 +373,7 @@
 	font-size: 12px;
 	line-height: 1;
 }
+
+.reader-stream-notice.notice.is-info {
```

Comment:
It's a good point. But I'm wondering if we have enough potential use cases for `update` status.